soc/projects/ml.md

```diff
@@ -147,53 +147,32 @@ simultaneously.
-an environment of varying complexity. 
-2. Contributed by industry and academicians in general has enriched its content over 
-past two decades and many of the proposed architectures are vetted time and again. The
-book has updated several times and each revision kind of presents itself as a 
-significant rewrite with all modern content. 
-3. The book is almost read by every AI engineer as a text for their graduation studies.
-So most people can connect to the content very easily. 
-
-**Why it's relevant to Julia?** - Here are some of the advantages of Julia:
-1. Most packages available today as AI packages focus on  the ML solvers as AI. If you 
-search the internet on the AI packages you will only get details of ML approaches used 
-today. Very few will talk about representational aspects of AI. 
-2. Julia with abstracted type systems and multiple dispatch interfaces provide an ideal 
+AIMA is a seminal text on representation of agents that can aid solve a problem for 
+an environment of varying complexity. Contributed by industry and academicians in 
+general has enriched its content over past two decades and many of the proposed 
+architectures are vetted time and again. The book has updated several times and each 
+revision kind of presents itself as a significant rewrite with all modern content. 
+The book is almost read by every AI engineer as a text for their graduation studies.
+So most people can connect to the content very easily. An AI framework developed 
+around the examples will provide a good foundation to design one's problem that can 
+utilize various non-ML or ML based solvers. 
+
+Most packages available today as AI libraries tend to focus on ML only and not the
+architectural aspect of AI. Very few will talk about representational aspects of AI. 
+Julia with abstracted type systems and multiple dispatch interfaces provide an ideal 
 language to represent the problem abstractions best as strategy patterns of software 
 engineering. So you can write the AIMA algorithms very easily as method with no side 
-effects in generic form. 
-3. Julia provides connectivity to other languages for using their libraries. Thus makes
-an ideal choice to represent the framework while can use solvers from other languages. 
-4. While AIMA authors have developed the sample code, their focus is more towards 
-developing sample code for teaching and learning and not professional usage of algorithms.
+effects in generic form. Julia provides connectivity to other languages for using their 
+libraries. Thus makes an ideal choice to represent the framework while can use solvers 
+from other languages. While AIMA authors have developed the sample code, their focus 
+is more towards developing sample code for teaching and learning and not professional 
+usage of algorithms.
 
 **Deliverables**
 
 The scope of the project is to create a library of core algorithms and sample programs 
```

Comment:
The deliverables usually aren't listed here since they are agreed upon directly between mentor and student given the students background and interests.